### ISSUE TITLE
Download fails fot free template  (Angular 17)

### ISSUE THREAD

**Issue by alexIslander** [DashboardPack/architectui-angular-theme-free#24]
When i go through the download process, the free template can't be downloaded.
Link throws Http 403 -> [LINK](https://dashboardpack.com/wp-content/uploads/woocommerce_uploads/2019/05/architectui-angular-free-theme.zip)

Please fix it.

**Comment by HICHEMBAKALEM** [comment]
I m facing the same issue 403

**Comment by vosecek** [comment]
Yep, no one reacts for months. Data are available trough github repository, but support seems to be bad, we consider buying Lifetime license but this brings some questions about support and future development.
